actioncable/test/subscription_adapter/postgresql_test.rb

```diff
@@ -12,7 +12,7 @@ def setup
     if Dir.exist?(ar_tests)
       require File.join(ar_tests, "config")
       require File.join(ar_tests, "support/config")
-      local_config = ARTest.config["arunit"]
+      local_config = ARTest.config["connections"]["postgresql"]["arunit"]
```

Comment:
As I understand, we want to use the same DB configuration as in ActiveRecord tests (that's why we're using `ARTest`).

`ARTest.config["arunit"]` is always `nil`, 'cause databases [config](https://github.com/rails/rails/blob/master/activerecord/test/config.example.yml) has nested structure (connections -> adapter -> id).

I'm using custom PostgreSQL credentials in `config.yml`, and it turned out that there is a bug here.